### PR TITLE
Package unicode-math

### PR DESCRIPTION
**Pull request by torik42**

### CODE REVIEW

File: yalafi/parameters.py [hunk 1]
```diff
@@ -393,9 +393,9 @@ def init_math_collections(self):
 
             '+', '-', '\\cdot', '\\times', '/',
             '=', '\\eq', '\\ne', '\\neq',
-            '<', '>', '\\le', '\\leq', '\\ge', '\\geq',
-            ':', ':=', '\\to', '\\cap', '\\cup',
-            '\\Leftrightarrow',
+            '<', '>', '≤', '≥', '\\le', '\\leq', '\\ge', '\\geq',
+            ':', ':=', '→', '\\to', '∩', '\\cap', '∪', '\\cup', '⇒', '\\Rightarrow', '⇐', '\\Leftarrow',
+            '⇔', '\\Leftrightarrow', '⊂',
```
Comment (torik42) [inline yalafi/parameters.py:398]: Is it okay to add those globally? So they apply even if `unicode-math` is not used.

Comment (matze-dd) [reply yalafi/parameters.py:398]: Maybe, you can extend the list in `init_module()`, similarly to the activation of macro `\text` in yalafi/packages/amsmath.py?

**Comment by matze-dd** [comment]
Thank you for this!

Hopefully this wish isn't impolite: could you please add the module to `yalafi/packages/__init__.py`? This is the default package list.

File: yalafi/parameters.py (hunk 1 above)
Comment (torik42) [reply yalafi/parameters.py:398]: Sure. Since I also added `'\\Rightarrow'`, shall this stay there?

**Comment by torik42** [comment]
> Hopefully this wish isn't impolite: could you please add the module to `yalafi/packages/__init__.py`? This is the default package list.

Sure. Could you maybe explain what is the reason of this list? I thought YaLafi loads all packages used via reading the `\usepackage{…}` commands. Is this only for `yalafi/tex2text.py`?

File: yalafi/parameters.py (hunk 1 above)
Comment (matze-dd) [reply yalafi/parameters.py]: Oh, yes. I didn't scroll the above code snipped to the right, so missed your \\Leftarrow and \\Rightarrow. These should best stay in the global list.

**Review by matze-dd — COMMENTED** [review]
Does `init_module()` work for you? I would expect
```
parms.math_operators.extend(math_operators)
```

**Comment by matze-dd** [comment]
> Could you maybe explain what is the reason of this list? I thought YaLafi loads all packages used via reading the \usepackage{…} commands. Is this only for yalafi/tex2text.py?

Yes, it is only used by tex2txt. By default, tex2txt loads all known packages (option --packages, placeholder \*). I thought, one might introduce other groups of packages later, if the number of packages becomes too large, and some structuring is necessary.

**Comment by torik42** [comment]
> Does `init_module()` work for you? I would expect
> 
> ```
> parms.math_operators.extend(math_operators)
> ```

You are right.  I only tested it with `≤` and `≥` which I forgot to delete from the default.

**Comment by matze-dd** [comment]
Many thanks, now we have it! I'll merge (just added a line to list-of-macros.md).

**Pull request merged**